pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -1220,6 +1227,71 @@ func isUSBNeeded(c *ConverterContext, vmi *v1.VirtualMachineInstance) bool {
 	return false
 }
 
+func setupDomainMemory(c *ConverterContext, vmi *v1.VirtualMachineInstance, domain *api.Domain) error {
```

Comment:
nit: Maybe a separate file for memory converter-related logic?

Reply:
just one function, can do but do you think it's necessary?

Reply:
Unused parameter 'c *ConverterContext'